README.md

```diff
@@ -93,24 +88,15 @@ We recommend using the latest versions of both docker images. Find them as GitHu
 
 
 ## Sync Check
+ 
+There is a sync check container. It fails at startup because at that point the replica is not running yet. But you can run it later with this command:
 
-To make sure your replica is running correctly, we've provided a script that checks its state roots against our sequencer.
-
-```
-yarn
-npx ts-node src/sync-check.ts
```

Comment:
oops sorry i misunderstood your commend! thought this was an older PR. i actually still run this myself since it's easier than starting the docker image, can we keep it?